src/SeedCppCore/Services/System/SystemServicesFactory.cpp

```diff
@@ -31,7 +33,9 @@ namespace seed_cpp { namespace service {
 
 	std::unique_ptr<systelab::rest_api_core::IEpochTimeService> SystemServicesFactory::buildEpochTimeService() const
 	{
-		return std::make_unique<systelab::rest_api_core::EpochTimeService>();
+		systelab::time::TimeAdapter TimeAdapter;
```

Comment:
After the change on REST API core to not require an epoch service to build a TokenExpirationAccessValidator, I think that this method is not used anywhere else. If that's the case, I would get rid of this factory method.